webapi/src/test/scala/org/knora/webapi/responders/v1/ResourcesResponderV1Spec.scala

```diff
@@ -206,7 +206,7 @@ object ResourcesResponderV1Spec {
             targetResourceIri = "http://data.knora.org/project-thing-2"
         )),
         occurrence = Some("0-n"),
-        attributes = "restypeid=http://www.knora.org/ontology/anything#Thing",
+        attributes = "size=45;restypeid=http://www.knora.org/ontology/anything#Thing",
```

Comment:
Since you defined a size for the property `anything:hasOtherThing`, this attribute's value has to be included in the expected test results.